src/main/java/pro/cloudnode/smp/nations/commands/NationsCommand.java

```diff
@@ -1,22 +1,422 @@
+        if (args.length <= 1) {
+            sendMessage(t(Messages.USAGE, label, "option", "<key> <value>"));
+            return;
+        }
+
+        Nation nation = Nations.getNationManager().getPlayerNation(getPlayer().getUniqueId());
+        if (nation == null) {
+            sendMessage(t(Messages.NOT_IN_NATION));
             return;
         }
 
+        if (!nation.leader.equals(getPlayer().getUniqueId())) {
+            sendMessage(t(Messages.NOT_LEADER));
+            return;
+        }
+
+        String key = args[1];
+
+        switch (key) {
+            case "color":
+                if (args.length == 2) {
+                    sendMessage(t(Messages.USAGE, label, "option", "color <color>"));
+                    return;
+                }
+                String color = args[2].toLowerCase();
+                boolean isHex = color.startsWith("#") && color.length() == 7 && color.matches("#[0-9a-fA-F]+");
+                if (!COLORS.contains(color) && !isHex) {
+                    sendMessage(t(Messages.INVALID_COLOR));
+                    return;
+                }
+
+                nation.color = color;
+                sendMessage(t(Messages.COLOR_SET, color));
+                nation.update();
+
+                break;
+            default:
+                sendMessage(t(Messages.USAGE, label, "option", "<key> <value>"));
+                break;
+        }
+    }
+
+    private void info(CommandSender sender, String label, String[] args) {
+        if (args.length > 1) {
+            sendMessage(t(Messages.USAGE, label, "info", ""));
+            return;
+        }
+
+        Nation nation = Nations.getNationManager().getPlayerNation(getPlayer().getUniqueId());
+        if (nation == null) {
+            sendMessage(t(Messages.NOT_IN_NATION));
+            return;
+        }
+
+        sendMessage(t(Messages.INFO_HEADER, nation));
+        sendMessage(t(Messages.INFO_LEADER, Bukkit.getPlayer(nation.leader).getName()));
+        sendMessage(t(Messages.INFO_MEMBERS, nation.members.stream().map(Bukkit::getPlayer).filter(Objects::nonNull).map(Player::getName).toList().toString()));
+    }
+
+    private void quit(CommandSender sender, String label, String[] args) {
+        if (args.length > 1) {
+            sendMessage(t(Messages.USAGE, label, "quit", ""));
+            return;
+        }
+
+        Nation nation = Nations.getNationManager().getPlayerNation(getPlayer().getUniqueId());
+        if (nation == null) {
+            sendMessage(t(Messages.NOT_IN_NATION));
+            return;
+        }
+
+        if (nation.members.isEmpty() && nation.leader.equals(getPlayer().getUniqueId())) {
+            sendMessage(t(Messages.NATION_DISBANDED, nation));
+            Nations.getNationManager().remove(nation);
+            return;
+        }
+
+        if (nation.leader.equals(getPlayer().getUniqueId())) {
+            sendMessage(t(Messages.CANT_QUIT_AS_LEADER));
+            return;
```

Comment:
btw to avoid have to multiline like
```java
if (…) {
    sendMessage(…);
    return;
}
```
I just make sendMessage return bool `true` (which we always want to return in commands anyways) and then make your execute also return bool and `if (…) return sendMessage(…)`. Makes things much simpler imo.